packages/esm-outpatient-app/src/patient-queue-metrics/clinic-metrics.component.tsx

```diff
@@ -43,7 +44,7 @@ const ClinicMetrics: React.FC = () => {
             onChange={handleServiceCountChange}
           />
         </MetricsCard>
-        <MetricsCard
+        <MetricsCardAvgWaitTime
```

Comment:
Thanks  @njorocs,  Average waiting time looks like a duplicated code of metrics component. we can simply add a prop to control when we need to show the `patient-list` link

Reply:
Thank you @donaldkibet. Let me revise the code with the suggested approach.